docs/manual/format.md

```diff
@@ -85,88 +88,72 @@ package, padded with null (0) bytes.
+built for.  In this case, 1 == Linux.  The next 2 bytes (78-79) form an int16
+that indicates the signature type.  This tells RPM what to expect in the
+Signature.  This is generally expected to be 5, which indicates the use of
+"Header-style" signatures.
 
 ```
 00000050: 04 00 00 00 68 e6 ff bf    ........
 00000058: ab ad 00 08 3c eb ff bf    ........
 ```
 
-The remaining 16 bytes (80-95) are currently unused and are reserved
-for future expansion.
+The remaining 16 bytes (80-95) are unused.
 
 ## Signature
 
-A 3.0 format signature (denoted by signature type 5 in the Lead), uses
-the same structure as the Header.  For historical reasons, this
-structure is called a "header structure", which can be confusing since
-it is used for both the Header and the Signature.  The details of the
-header structure are given below, and you'll want to read them so the
-rest of this makes sense.  The tags for the Signature are defined in
-lib/signature.h.
-
-The Signature can contain multiple signatures, of different types.
-There are currently only three types, each with its own tag in the
-header structure:
-
-```
-	Name	Tag	Header Type
-	----	----	-----------
-	SIZE	1000	INT_32
-	MD5	1001	BIN
-	PGP	1002	BIN
-```
-
-The MD5 signature is 16 bytes, and the PGP signature varies with
-the size of the PGP key used to sign the package.
-
-As of RPM 2.1, all packages carry at least SIZE and MD5 signatures,
-and the Signature section is padded to a multiple of 8 bytes.
+"Header-style" signatures (denoted by signature type 5 in the Lead), use the
+same structure as the Header.  For historical reasons, this structure is called
+a "header structure", which can be confusing since it is used for both the
+Header and the Signature.  The details of the header structure are given below,
+and you'll want to read them so the rest of this makes sense.  The tags for the
+Signature are defined in include/rpm/rpmtag.h.
+
+The Signature can contain multiple different types of signatures, stored under
+unique tags (just like the Header). Details about these tags and the information
+they store can be found [here](signatures_digests.md).
+
+RPM v4 packages are expected to contain at least one of SHA1HEADER or SHA256HEADER
+tags, providing a cryptographic digest of the main header, and may contain one
+or both of the PAYLOADDIGEST and PAYLOADDIGESTALT tags, providing a cryptographic
+digest of the package payload in the compressed and uncompressed forms, respectively.
+
+If the package has been cryptographically signed using OpenPGP, an RSAHEADER or
+DSAHEADER tag ought to be present, which contains an OpenPGP signature of the
+package header. Which tag is present depends on which of the two (supported)
+OpenPGP algorithms was used at signing time. Using a key based upon the RSA
+algorithm to sign the package will result in the signature being stored in the
+RSAHEADER tag, whereas the use of the EdDSA (ed25519) algorithm will use the
```

Comment:
I'm not a cryptography person, I'm unsure if this is an appropriate way to refer to an EdDSA signature that uses curve ed25519, or if it's OK to just refer to it as EdDSA (as happens in a few other places)